modules/chapter4/pages/section4.adoc

```diff
@@ -1,6 +1,191 @@
 
-== Prerequisites
\ No newline at end of file
+== Prerequisites
+
+. Verify that _tenant_ cluster is deployed successfully.
+
+. Access the _tenant_ cluster via CLI and web console.
+
+. Ensure that all nodes are in `Ready` status and all cluster operators are available.
++
+.Sample output:
+----
+[root@hypervisor ~]# oc get nodes
+NAME                   STATUS     ROLES                         AGE   VERSION
+tcn1.lab.example.com   Ready      control-plane,master,worker   15h   v1.29.7+4510e9c
+tcn2.lab.example.com   NotReady   control-plane,master,worker   14h   v1.29.7+4510e9c
+tcn3.lab.example.com   Ready      control-plane,master,worker   15h   v1.29.7+4510e9c
+
+[root@hypervisor ~]# oc get clusterversion
+
+NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.16.8    True        False         23h     Cluster version is 4.16.8
+----
+
+== Install OpenShift Data Foundation Operator
+
+. Access the operator hub from the web console of _tenant_ cluster.
++
+From the left navigation pane, click menu:Operators[OperatorHub].
++
+image::tenant_console_operator_hub.png[]
+
+. In a search window, search _odf_ and select the `OpenShift Data Foundation`.
++
+image::tenant_console_odf_install.png[]
+
+. Click btn:[Install] to open install options.
++
+image::tenant_console_odf_install_1.png[]
+
+. Keep all options as is, with no change, in selected options and then click btn:[Install] to install the operator.
++
+image::tenant_console_odf_install_2.png[]
+
+. After 3-4 minutes, notice the `Refresh web console` message on window.
++
+First, refresh the web console and then click btn:[Create StorageSystem] to create the resource.
++
+image::tenant_console_odf_install_3.png[]
+
+. Create the StorageSystem.
+
+.. Provide backing storage details.
++
+Deployment type: `Full deployment`.
++
+Select `Connect an external storage platform`.
++
+Storage platform: `Red Hat Ceph Storage`.
++
+image::tenant_console_odf_install_4.png[]
++
+Check box the `Use Ceph RBD as the default StorageClass`.
++
+Click btn:[Next] to proceed.
++
+image::tenant_console_odf_install_5.png[]
+
+.. Click btn:[Browse] to provide the content of the `output.json` file.
++
+image::tenant_console_odf_install_7.png[]
+
+.. Select the `output.json` file and click btn:[Open].
++
+image::tenant_console_odf_install_8.png[]
+
+.. Click btn:[Next] to proceed.
++
+image::tenant_console_odf_install_9.png[]
+
+.. Click btn:[Create StorageSystem] to create the StorageSystem.
++
+image::tenant_console_odf_install_10.png[]
+
+.. Notice the raw and used capacity.
++
+image::tenant_console_odf_install_11.png[]
+
+.. Notice the conditions as `Available, VendorCsv Ready, Vendor System Present`.
++
+image::tenant_console_odf_install_12.png[]
+
+.. Verify that the operator is installed successfully in `openshift-storage` namespace.
++
+image::tenant_console_odf_install_13.png[]
+
+== Configure the Image Registry Operator to use Noobaa storage with Red Hat OpenShift Data Foundation
```

Comment:
Add a period at the end of this sentence.

Reply:
?